scripts/training.py

```diff
@@ -303,6 +292,37 @@ def optimize_model():
 
 env.close()
 
-# (neptune) Log model weights
+# (Neptune) Initialize a new model version
+from neptune.new.exceptions import NeptuneModelKeyAlreadyExistsError
+
+with contextlib.suppress(NeptuneModelKeyAlreadyExistsError):
+    # Initialize a new model if it doesn not already exist
+    model = neptune.init_model(name="cartpole", key="CART")
+
+model_version = neptune.init_model_version(model="PROJRL-CART", name="cartpole")
+
+# (Neptune) Log model weights to model version
 torch.save(policy_net.state_dict(), "policy_net.pth")
-run["agent/policy_net"].upload("policy_net.pth")
+model_version["weights"].upload("policy_net.pth")
+
+# (Neptune) Associate run with model version and vice-versa
+run_meta = {
+    "id": run.get_structure()["sys"]["id"].fetch(),
+    "name": run.get_structure()["sys"]["name"].fetch(),
+    "url": run.get_url(),
+}
+model_version["run"] = run_meta
+
```

Comment:
Client should provide these helpers in utils for something.

Reply:
yeah, this is in the roadmap: https://portal.neptune.ai/c/130-associate-model-versions-with-runs